front-end/src/components/optionSquare/optionSquare.tsx

```diff
@@ -0,0 +1,28 @@
+import { useState } from 'react';
+import styles from './optionSquare.module.scss';
+import cx from 'classnames';
+
+interface OptionSquareProps {
+  label: string;
+  illustration: string;
+  clicked: boolean;
+  setClicked: (clicked: boolean) => void;
+}
+
+export const OptionSquare = ({ label, illustration, clicked, setClicked}: OptionSquareProps) => {
+    
+    const handleClick = () => {
+        setClicked(!clicked);
+    };
+
+    return (
+        <div className={cx(styles.optionSquare, { [styles.clicked]: clicked })} onClick={handleClick}>
+        <div className={styles.illustration}>
+                {illustration ? <img src={illustration} alt="Illustration" /> : null}
```

Comment:
Now that illustration is a required field you should be able to get rid of this I think
```suggestion
                <img src={illustration} alt="Illustration" />
```